tests/unittest/_torch/attention/sparse/dsa/test_fused_metadata.py

```
@@ -0,0 +1,365 @@
+    return req_idx, slot_fp8, slot_scale, gen_kv_indptr, gen_cached
+
+
+def _assert_bit_identical(fused, eager):
+    names = [
+        "req_idx_per_token",
+        "slot_mapping_fp8",
+        "slot_mapping_scale",
+        "gen_kv_indptr",
+        "gen_cached_token_indptr",
+    ]
+    for name, f, e in zip(names, fused, eager):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)